Example/StripeConnectExample/StripeConnectExample/Helpers/AppearanceFromURL.swift

```diff
@@ -0,0 +1,192 @@
+//
+//  AppearanceFromURL.swift
+//  StripeConnect Example
+//
+//  Created by Chris Mays on 9/5/24.
+//
+
+import Foundation
+import UIKit
+@_spi(PrivateBetaConnect) import StripeConnect
+
+typealias Appearance = EmbeddedComponentManager.Appearance
+
+extension Appearance {
+    static func fromURL(_ urlString: String) -> Appearance? {
+        guard let url = URL(string: urlString),
+              let components = URLComponents(url: url, resolvingAgainstBaseURL: false),
+              let initOptionsOverride = components.queryItems?.first(where: { $0.name == "initOptionsOverride" })?.value,
+              let data = initOptionsOverride.removingPercentEncoding?.data(using: .utf8),
+              let json = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: Any],
+              let appearance = json["appearance"] as? [String: Any] else {
+            return nil
+        }
+        return createAppearance(from: appearance)
+    }
+
+    private static func createAppearance(from dict: [String: Any]) -> Appearance? {
```

Comment:
Instead of encoding this into the example app, I think it would make more sense to put into our docs site. See go/grep/generateInitScript for our JS appearance generation code -- we could make a similar one for Swift.